config/packages

```diff
@@ -35,6 +35,6 @@ ttf-bitstream-vera
 unrtf
 unzip
 uuid-dev
-wkhtmltopdf-static
+wkhtmltox
```

Comment:
This commit should also change `HTML_TO_PDF_COMMAND` in `config/general.yml-example`